apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/Wallet.java

```diff
@@ -0,0 +1,54 @@
+package com.apollocurrrency.aplwallet.inttest.model;
+
+public class Wallet {
```

Comment:
I suggest to use lombook annotations here.

Reply:
Done